CONTRIBUTING.md

```diff
@@ -149,14 +149,33 @@ Release Process
         ```
     1. Ensure no commits are made on ni/nimi-python/master until the release is complete
     1. Create and checkout a branch for release-related changes
+    1. Perform Version Bump (If Needed)
+        * If you need to upgrade the major or minor versions, include any of the following parameters:
+           * --increment-major-version - To increment the major version of package. This will update the version to (N+1).X.X.dev0
+           * --increment-minor-version - To increment the minor version of package. This will update the version to X.(N+1).X.dev0
+            For example: 
+                ```bash
+                python3 tools/build_release.py --increment-minor-version
+                ```
```

Comment:
```suggestion
           * Example: `python3 tools/build_release.py --increment-minor-version`
```

Markdown treats that "For example" as part of the previous line when rendering.()